src/test/java/laddergame/domain/HeightTest.java

```diff
@@ -0,0 +1,36 @@
+package laddergame.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+
+@DisplayName("높이")
+class HeightTest {
+
+    @DisplayName("높이가 0이거나 음수이면 예외가 발생한다.")
+    @ParameterizedTest
+    @ValueSource(ints = {-1, 0})
+    void throwExceptionWhenValueIsZeroOrNegative(final int value) {
+        assertThatThrownBy(() -> new Height(value));
+    }
```

Comment:
간단한 테스트라도
//given
//when
//then 을 적용해보시는건 어떨까요?

```suggestion
    void throwExceptionWhenValueIsZeroOrNegative(final int value) {
        //given
        //when
        //then
        assertThatThrownBy(() -> new Height(value));
    }
```

Reply:
어떻게.. 나눠야 할지 모르겠습니다 ... 아니면 이미 나눠주신걸까요?

Reply:
넵. 아래처럼 나눠보는건 어떨지 제안드린 부분이었어요.